### ISSUE TITLE
Version: 1.4.0 breaks native payment methods

### ISSUE THREAD

**Issue by thompalstra** [paynl/shopware6-plugin#171]
In version Version: 1.4.0 you guys have added a "save" button to each payment method to prevent selection of a method to be directly applied (without banking data). This however breaks ALL native functionality.

Native payment methods (added via the administration) do not have a "save" button thus cannot be applied. Besides this, Shopware already supports a perfectly fine method for applying payment methods (ajax, then refresh due to certain rules enabling or disabling shipping methods based on conditions for on for example payment methods).

I'm not really sure what you guys are thinking, but the Javascript is an immense mess to.

Take this for example (which also breaks all the time):

````
    const currentPaymentBlock = event.target.parentNode.parentNode.parentNode; // what?
    const extraDataBlock = currentPaymentBlock.querySelector('.paynl-payment-method-extra');

    const issuerSelect = extraDataBlock.querySelector('.paynl-ideal-banks-select'); // ok, this will be a huge issue
````

Who in their right mind just bubbles up a x-numbers of parents and just uses a queryselector for matching. After this message, you guys just happily use a querySelector on ``extraDataBlock`` without even checking if that element exists?

```
    const xhr = new XMLHttpRequest();
    xhr.open('POST', '/PaynlPayment/order/change/paylater-fields', true);
    xhr.setRequestHeader('Content-Type', 'application/json');
    xhr.send(JSON.stringify(data));
```

Please don't use xhr directly. Shopware supports API calls inheritely if registrered correctly. This way you guys are digging your own grave.

Can you guys PLEASE revert to using the default shopware functionality in stead of reinventing the wheel? For bank selection you guys "could" use the ``onChangeCallback`` but for other methods that have no additional fields / settings please leave the original functionality intact.

**Comment by ahudyma** [comment]
Hello. We will check the reported issue. However, if you are about to continue this communication try to stay more polite. An offensive attitude never works. And of course, if there is any knowledge to share, a pull requested is always welcomed.

**Comment by ahudyma** [comment]
The bug was fixed in 1.4.2
